### PR TITLE
Add Contributing and Code of Conduct

### PR DESCRIPTION
**Pull request by vdutor**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement (the [project maintainers](CONTRIBUTING.md#who-are-we?)) by private message on the [Slack workspace](https://secondmind-labs.slack.com).
```
Comment (vabor112) [inline CODE_OF_CONDUCT.md:67]: Private message to Secondmind Slack? Maybe this should be deleted?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+https://www.contributor-covenant.org/faq. Translations are available at
+https://www.contributor-covenant.org/translations.
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
```
Comment (vabor112) [inline CODE_OF_CONDUCT.md:140]: With this line starts the copy-paste of the above.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Contribution guidelines
+
+### Who are we?
+
+*Maintainers* (in alphabetical order: Viacheslav Borovitskiy, Vincent Dutordoir, Michael Hutchinson, Noemie Jaquier,  Peter Mostowsky, Aditya Ravuri, Leonel Rozo, Alexander Terenin) steer the project, keep the community thriving, and manage contributions.
```
Comment (vabor112) [inline CONTRIBUTING.md:5]: Maybe we should discuss the list in Slack, check if everybody wants to be in this list and is ready to carry some responsibility associated to that.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+### Reporting a bug
+
+Finding and fixing bugs helps us provide robust functionality to all users. You can either submit a bug report or, if you know how to fix the bug yourself, you can submit a bug fix. We gladly welcome either, but a fix is likely to be released sooner, simply because others may not have time to quickly implement a fix themselves. If you're interested in implementing it, but would like help in doing so, you can send [the maintainers](#who-are-we) an email or open an [issue](https://github.com/GPflow/GeometricKernels/issues/new).
+
+We use GitHub issues for bug reports. You can use the [issue template](https://github.com/GPflow/GeometricKernels/issues/new) to start writing yours. Once you've submitted it, the maintainers will take a look as soon as possible, ideally within the week, and get back to you about how to proceed. If it's a small easy fix, they may implement it then and there. For fixes that are more involved, they will discuss with you about how urgent the fix is, with the aim of providing some timeline of when you can expect to see it.
```
Comment (vabor112) [inline CONTRIBUTING.md:15]: We don't seem to have those.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+If you'd like to submit a bug fix, the [pull request templates](https://github.com/GPflow/GeometricKernels/compare) are a good place to start. We recommend you discuss your changes with the community before you begin working on them, so that questions and suggestions can be made early on.
```
Comment (vabor112) [inline CONTRIBUTING.md:17]: The link seems to lead to a strange destination

**Review by stoprightthere — APPROVED** [review]
LGTM

**Pull request merged**